packages/design-system/src/atoms/inputV2/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { forwardRef, useCallback, useMemo, useState } from 'react'
+import { StyleSheet, TextInput, TextInputProps, View } from 'react-native'
+import { useThemeV2 } from '../../hooks'
+import { container, fontStyles, margin, padding, spacing } from '../../styles'
+import { Icon, IconVariant } from '../icon'
+import { TypographyV2 } from '../typographyV2'
+
+export interface InputPropsV2 extends TextInputProps {
+  iconVariantRight?: IconVariant
+  iconVariantLeft?: IconVariant
+  error?: string
+  label?: string
+  withCount?: boolean
```

Comment:
It should be more API-friendly :)

```suggestion
  counter?: boolean
```